.github/workflows/pregenerate.yml

```diff
@@ -0,0 +1,39 @@
+name: Pregenerate Files
```

Comment:
If the purpose of this workflow is to _check_ the generated files and not actually generate them, it should be more indicative here.

Reply:
Renamed to `Check Pregenerated Files`